packages/integration-tests/api/api.js

```diff
@@ -10,6 +11,29 @@ const {
   tokenUtils: { createJwtToken }
 } = require('@cumulus/api');
 
+function invokeApi(prefix, payload) {
+  return pRetry(
```

Comment:
From what I can find on the docs the default retry # is `10`. Would it make sense to make the retry value configurable as an input to the `invokeApi` function?